.github/workflows/docker.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish (docker-image)
```

Comment:
why docker change in this pr?

Reply:
I guess it was a github glitch, its looks like it corrected itself.